doc/source/conf.py

```diff
@@ -162,7 +162,6 @@ def find_test_modules(package_path):
     # Renamed pages
     "installation": "how-to-install-flower.html",
     "configuring-clients.html": "how-to-configure-clients.html",
-    "quickstart_mxnet": "tutorial-quickstart-mxnet.html",
```

Comment:
@charlesbvll In case we would instead of removing redirect these to the docs landing page it would be the following I believe:

```
"quickstart_mxnet": "index.html",
```

Can you comment on that? I might be wrong.

Reply:
@tanertopal Yes, that would work, we should also add the redirect for the current url:
```
    "quickstart_mxnet": "index.html",
    "quickstart-mxnet": "index.html",
```